src/com/facebook/buck/features/apple/projectV2/FlagParser.java

```diff
@@ -195,6 +199,27 @@ void parseFlags(
                     targetNode.getBuildTarget(),
                     "swift_include_paths",
                     searchPathAttributes.swiftIncludeFlags()));
+
```

Comment:
Please add a comment here explaining (briefly) why this is necessary, something like:

```java
// Xcode relies on VSFOverlay file for indexing to work, and this file can't be cached because contains absolute paths.
// We have to generate it here otherwise remote caching breaks indexing.
```

I would also like it if you add a description to the PR.

So we can understand why this change was necessary for the future.